homeassistant/components/esphome/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "ESPHome",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/esphome",
-  "requirements": ["aioesphomeapi==13.1.0", "esphome-dashboard-api==1.1"],
+  "requirements": ["aioesphomeapi==13.1.0", "esphome-dashboard-api==1.2.1"],
```

Comment:
There's no link to the release of esphome-dashboard-api in the PR description. I tried to find it but I only found the PyPI project not the github project. The PyPI project just links to the Home Assistant frontend project.

https://pypi.org/project/esphome-dashboard-api/

Can we clarify where the library can be found, with releases etc, and link appropriately? Thanks!

Reply:
https://github.com/esphome/dashboard-api

Will update the pypi links.

Reply:
And updated description to point at https://github.com/esphome/dashboard-api/releases/tag/1.2.1

Reply:
Thanks!